.circleci/test-example-repo.sh

```diff
@@ -4,8 +4,25 @@ set -ex
 
 TERMINUS_SITE=build-tools-$CIRCLE_BUILD_NUM
 
-BRANCH=$(echo $CIRCLE_BRANCH | grep -v '^\(master\|[0-9]\+.x\)$')
-PR_BRANCH=${BRANCH:+dev-$BRANCH}
+# If we are on the master branch
+if [[ $CIRCLE_BRANCH == "master" ]]
+then
+    PR_BRANCH="dev-master"
+else
+    # If this is a pull request use the PR number
+    if [[ -z "$CIRCLE_PULL_REQUEST" ]]
+    then
+        # Stash PR number
+        PR_NUMBER=${CIRCLE_PULL_REQUEST##*/}
```

Comment:
Odd, I re-ran the build with SSH and just tested this section and it was working properly.

`CIRCLE_PULL_REQUEST` return the pull request URL, `PR_NUMBER=${CIRCLE_PULL_REQUEST##*/}` just strips everything up to and including the last dash which leaves the pull request number.

![screen shot 2017-07-19 at 10 47 46 am](https://user-images.githubusercontent.com/2133004/28376553-0235a666-6c70-11e7-95f2-7e06a04ba918.png)

Should `BUILD_TOOLS_VERSION` just return `1.x` or `2.x`?